locustempus/main/tests/test_permissions.py

```diff
@@ -197,3 +201,238 @@ def test_has_object_permissions_delete(self):
         self.assertFalse(
             perm.has_object_permission(
                 req, None, self.sandbox_course_response))
+
+
+class IsFeedbackFacultyOrStudentRecipientTest(CourseTestMixin, TestCase):
+    def setUp(self):
+        self.setup_course()
+
+    def test_has_permissions_list_get(self):
+        """Tests GET list permissions"""
+        perm = IsFeedbackFacultyOrStudentRecipient()
+        req = RequestFactory().get(
+            reverse('api-feedback-list')
+        )
+        req.query_params = dict()
+        req.data = dict()
+
+        # Anon
+        req.user = AnonymousUser()
+        self.assertFalse(perm.has_permission(req, None))
+
+        # Faculty
+        req.user = self.faculty
+        self.assertFalse(perm.has_permission(req, None))
+
+        # Student
+        req.user = self.student
+        self.assertFalse(perm.has_permission(req, None))
+
+    def test_has_permissions_list_get_w_querystring(self):
+        """Tests GET list permissions"""
+        perm = IsFeedbackFacultyOrStudentRecipient()
+        req = RequestFactory().get(
+            reverse('api-feedback-list') + '/?activity={}'.format(
+                self.sandbox_course_activity.pk))
+        req.query_params = {'activity': self.sandbox_course_activity.pk}
+        req.data = dict()
+
+        # Anon
+        req.user = AnonymousUser()
+        self.assertFalse(perm.has_permission(req, None))
+
+        # Faculty
+        req.user = self.faculty
+        self.assertTrue(perm.has_permission(req, None))
+
+        # Student
+        req.user = self.student
+        self.assertTrue(perm.has_permission(req, None))
```

Comment:
Classmate might not make sense here because they should have permission too. They should be able to see their own feedback.

I did add a schoolmate test, which tests that a student in an another course can't see feedback for this course.

Reply:
Ahh...okay...that makes sense.